cmd/minishift/cmd/start.go

```diff
@@ -349,9 +345,10 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 	if err != nil {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error starting the VM: %v", err))
 	}
-	spinnerView.Stop()
-	fmt.Println("OK")
+	// Stops the progress of dots
+	progressDots <- true
```

Comment:
`progressDots <- true` should be in a function. Even if it is just one line a function would help to encapsulate it and if we want to change or add more to it in the future it would be easy.

Reply:
Agree... abstract it.

Maybe even as a package for just the progress dots, so it reads as
```
progress.Start()
...
proogress.Stop()
```